lib/entities/npcs/animals/piggy.dart

```diff
@@ -125,6 +125,8 @@ class Piggy extends NPC {
 		SkillManager.learn(SKILL, email);
 		nibbleCounts[email] = (nibbleCounts[email] ?? 0) + 1;
 
```

Comment:
You'll need to add a line here to actually make this event count towards their active quest log:

messageBus.publish(new RequirementProgress('piggyNibble', email));

This basically sends a message which will be received by a quest listener for the user matching "email". It doesn't much matter what you call "piggyNibble" so long as it's a unique string among all quest requirements and that it matches (via regex) the eventType in the quest json.